### DISCUSSION TITLE
I need a application to change any WiFi password only

### DISCUSSION THREAD

**Discussion by Nexus123456789** [AppIntro/AppIntro#983] [Q&A] [answered]
Hi.. I am new to programming and after 10 days..I am going to participate on my first Hackathon so I need app that takes IP address (i.e. 192.178.0.1) then take username and password and after that there is only one option i.e to change WiFi password only... So can I make that in one day or not.. and I need help to make such application

**Comment by paolorotolo** [comment, marked as answer]
Hey @Nexus123456789! This discussion space is reserved to the development of the Android Library "AppIntro". I'm closing this for off topic. 

Good luck with your hackathon! :)
